src/app/components/atoms/Map/map.tsx

```diff
@@ -0,0 +1,108 @@
+'use client'
+import React from 'react'
+import { GoogleMap, MarkerF, useJsApiLoader , InfoWindowF} from '@react-google-maps/api';
+
+interface Place {
+  lat: number,
+  lng: number,
+  name: string,
+}
+
+interface MapProps {
+  center: Place,
+  places: Place[],
+}
+
+export default function MapAtom({ center, places }: MapProps) {
+  const [map, setMap] = React.useState(null)
+  const [place, setPlace] = React.useState(center)
+
+  const { isLoaded } = useJsApiLoader({
+    id: 'google-map-script',
+    googleMapsApiKey: "AIzaSyA2oBnN5Vzw8NDlZO7O6PlKrdY_OPXNLbY"
+  })
+
+  const onLoad = React.useCallback(function callback(map: any) {
+    setMap(map)
+  }, [])
+  
+  const onUnmount = React.useCallback(function callback(map: any) {
+    setMap(null)
+  }, [])
+  
+  const containerStyle = {
+    width: '100%',
+    height: '500px',
+    borderRadius: '0.5rem',
+  
+  };
+
+  return isLoaded ? (
+      <div>
+        <GoogleMap
+          className='rounded-lg'
+          mapContainerStyle={containerStyle}
+          center={center}
+          zoom={13}
+          onLoad={onLoad}
+          onUnmount={onUnmount}
+        >
+          { /* Center marker */ }
+          <MarkerF
+            icon={'/images/redmarker.svg'}
+            title={center.name}
+            position={center}
+            onClick={() => 
+              setPlace({ lat: center.lat, lng: center.lng , name: center.name})
+            }
+          />
+          { /* Other markers */}
+          {places.map((place,i) => (
+            <div>

```

Comment:
Falta agregar un key, marca error React cuando haces un map de un error si no pones algún key

```suggestion
            <div key={`marker-${i}`}>
```